server/test/controllers/applicant/EligibilityAlertSettingsCalculatorTest.java

```diff
@@ -0,0 +1,166 @@
+package controllers.applicant;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+import static play.test.Helpers.fakeRequest;
+
+import auth.ProgramAcls;
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import controllers.FlashKey;
+import java.util.Collections;
+import java.util.Map;
+import java.util.Optional;
+import junitparams.JUnitParamsRunner;
+import junitparams.Parameters;
+import models.DisplayMode;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import play.i18n.Lang;
+import play.i18n.Langs;
+import play.i18n.MessagesApi;
+import play.mvc.Http;
+import services.AlertSettings;
+import services.AlertType;
+import services.MessageKey;
+import services.program.ProgramDefinition;
+import services.program.ProgramNotFoundException;
+import services.program.ProgramService;
+import services.program.ProgramType;
+import services.program.StatusDefinitions;
+
+@RunWith(JUnitParamsRunner.class)
+public class EligibilityAlertSettingsCalculatorTest {
+  private MessagesApi getMessagesApiMock() {
+    Langs langs = new Langs(new play.api.i18n.DefaultLangs());
+
+    Map<String, String> messagesMap =
+        ImmutableMap.<String, String>builder()
+            .put(
+                MessageKey.ALERT_ELIGIBILITY_APPLICANT_FASTFORWARDED_ELIGIBLE_TITLE.getKeyName(),
+                "A11")
+            .put(
+                MessageKey.ALERT_ELIGIBILITY_APPLICANT_FASTFORWARDED_ELIGIBLE_TEXT.getKeyName(),
+                "A12")
```

Comment:
Question - where does the values like `A12` `A21` come from? Are they just fake data for testing?